lib/omniauth/strategies/openid_connect.rb

```diff
@@ -313,14 +310,29 @@ def id_token_callback_phase
       end
 
       def valid_response_type?
-        return true if params.key?(options.response_type)
+        return true if params.key?(response_type)
 
-        error_attrs = RESPONSE_TYPE_EXCEPTIONS[options.response_type.to_s]
+        error_attrs = RESPONSE_TYPE_EXCEPTIONS[response_type]
         fail!(error_attrs[:key], error_attrs[:exception_class].new(params['error']))
 
         false
       end
 
+      def response_type
+        @response_type ||= options.response_type.to_s
```

Comment:
```ruby
[2] pry(main)> ['id_token', 'code'].to_s
=> "[\"id_token\", \"code\"]"
```

I don't think that it's a good response_type. It should be like: `id_token,code`

Reply:
valid response type params are separated by single space character, e.g. `id_token token`

Reply:
Oh, sorry, that's true! Too many standards!

Reply:
We need to make a clear distinction between this gem's setting `options.response_type` which, from what I see, is not yet determined for multiple response types and the OAuth2 standard for response_type (which is clearly defined: one or multiple values, separated with spaces).